### ISSUE TITLE
PackedBubble Series Not Honoring lang.thousandsSep

### ISSUE THREAD

**Issue by wergeld** [highcharts/highcharts#10062]
#### Expected behaviour
Using setOptions to configure comma as the thousands separator and setting tooltip.pointFormat should also work in packedbubble series as it does for other series.

#### Actual behaviour
Thousands separator is ignored.

#### Live demo with steps to reproduce
https://jsfiddle.net/wergeld/f0gxpjbt/

Hover over the larger circles with values greater than 999. Notice no comma used for thousands.

#### Product version
Highcharts 7.0.2

#### Affected browser(s)
Appears to affect all/

**Comment by Izothep** [comment]
Hi @wergeld!

Thank you for reporting your issue. It looks that the problem is related to using point.y instead of point.value in tooltip pointFormat.

When using point.value the problem is not visible anymore. Live example:
https://jsfiddle.net/izothep/s1korfm3/1/

**Comment by wergeld** [comment]
@Izothep, This worked! Thank you very much. Is this the expected point format though?

**Comment by pawelfus** [comment]
Actually, it looks like `point.y` accidentally works in tooltip. There's no `point.y` for packed bubble series - only `point.value`, as in the config in `data`. 

Longer explanation:
We internally add `:.,f` to each `{point.something}` changing to `{point.something:.,f}` if `something` is part of the official API. For example `y` is not listed in the options for a bubble point: https://api.highcharts.com/highcharts/series.packedbubble.data (PS: @Izothep `value` is also missing from the tree on the left side navigation ;) ) But for an area, we have access to `x` and `y` as valid formats: https://api.highcharts.com/highcharts/series.area.data - which we do automatically (even more complex solution for date-formats). The list of elements with automated formats (and much more) is available for a dev through `Highcharts.seriesTypes.packedbubble.prototype.pointArrayMap` path.

In other words, `{point.y}` does require full format `{point.y:.,f}` as any other custom option. Whenever possible `{point.value}` should be used, just like in the API: https://api.highcharts.com/highcharts/series.packedbubble.tooltip.pointFormat

**Comment by pawelfus** [comment]
`{point.value}` does honor the lang settings, see demo by @Izothep : https://jsfiddle.net/izothep/s1korfm3/1/

**Comment by wergeld** [comment]
@pawelfus 
So, the original highcharts provided demo using point.y was incorrect? I just want to make sure I can continue to use point.value going forward for packed bubble charts.

**Comment by pawelfus** [comment]
Wow, yes, in my opinion it's wrong :) Thanks for pointing this out! @Izothep
